app/src/main/java/com/lagradost/cloudstream3/services/VideoDownloadService.kt

```diff
@@ -34,13 +44,19 @@ class VideoDownloadService : Service() {
             }
         }
 
+        VideoDownloadManager.startServiceForeground(this)
```

Comment:
The video download manager is already running using the worker foreground service:
```
Started foreground service Intent { act=ACTION_START_FOREGROUND cmp=com.lagradost.cloudstream3.prerelease.debug/androidx.work.impl.foreground.SystemForegroundService (has extras) }
```
Starting another service (without proper termination) is unnecessary

Reply:
I will remove personally due to the urgency of a fix